src/knot_protocol.h

```diff
@@ -87,40 +87,34 @@ typedef struct __attribute__ ((packed)) {
 	uint8_t			payload_len;
 } knot_msg_header;
 
-typedef struct __attribute__ ((packed)) {
-	uint8_t			sensor_id;	// App defined sensor id 
-	uint8_t			unit;		// KNOT_UNIT_* 
-	uint8_t			value_type;	// KNOT_VALUE_TYPE_* (int, float, bool, raw)
-} knot_data_header;
-
```

Comment:
All these changes will break GW code. Can you please split this patchset and send both series together? (changes at protocol & GW)